proto/sedachain/tally/v1/tally.proto

```diff
@@ -15,7 +15,8 @@ message Params {
   // FilterGasCostMultiplierStdDev is the gas cost multiplier for a filter type
   // stddev.
   uint64 filter_gas_cost_multiplier_std_dev = 4;
-  // GasCostCommitment is the gas cost for a commitment corresponding to an
-  // expired data request.
-  uint64 gas_cost_commitment = 5;
+  // GasCostBase is the base gas cost for a data request.
+  uint64 gas_cost_base = 5;
+  // GasCostCommit is the gas cost for a commit charged under certain scenarios.
+  uint64 gas_cost_commit = 6;
```

Comment:
where is this used?

Reply:
We charge the base gas cost for every data request.

When the request times out or when there is no basic consensus, we charge the commit cost for each executor that did commit (`CalculateCommitterPayouts()`)

Reply:
Having given it some though maybe it's good to have the name be a little more descriptive of when it's used? I feel like that current name implies that it's a base fee we're charging for the commitment, but in reality it's the fallback gas consumption when we have no other way of determining how much gas was used.

Reply:
agreed and changed the name to `GasCostFallback`